openc3/data/config/tool.yaml

```diff
@@ -82,3 +82,15 @@ TOOL:
           required: false
           description: Regex to match against filenames. If match, then no ERB processing
           values: .+
+    IMPORT_MAP_ITEM:
+      summary: Add an item to the import map
```

Comment:
Can I get a `description` with a little more information about when and why this is needed?
"Needed for custom tools because ..."

Reply:
Ok ... let's remember to update the docs when we get to that point!